jenkins_jobs/update_versions_readme/script.sh

```diff
@@ -0,0 +1,9 @@
+python host-os.py \
+  --verbose \
+  update-versions-readme \
```

Comment:
Your point makes sense, but can't we update the versions of *all* packages in the README and update the versions of a sublist of packages in yamls in upgrade-versions? IMHO it is better to automatically update README whenever the yamls are updated to avoid extra work and accidental inconsistencies.